lib/pbench/client/__init__.py

```diff
@@ -208,7 +205,7 @@ def put(
         api: API,
         uri_params: Optional[JSONOBJECT] = None,
         *,
-        json: Optional[dict[str, str]] = None,
+        json: Optional[dict[str, Any]] = None,
```

Comment:
Just a thought:  the type `JSONOBJECT` was good at line 206...perhaps we should consider using it at line 208 for the _`json`_ parameter....  😁

Ditto at line 243.